src/Entity/Menu.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Entity;
+
+class Menu extends Element
+{
+
+    private $quantite;
```

Comment:
you extend Element which already has quantite and price.
The problem is that in Element, the properties have been set private. therefor the cannot be used in the children. ❗️